__tests__/index.test.js

```diff
@@ -0,0 +1,173 @@
+import 'jest-styled-components';
```

Comment:
I think this test suit is testing more than it should be.

In a nutshell, creating react components and check if the styles were applied to them is a task for styled component itself, since we're using a method from them.

The library:

1. returns an object with some methods already configured;
2. each function of this is calling `styled-components/css` tag template literals with a string

In that sense, it might be a cleaner implementation:

1. remove all React stuffs;
2. mock `css` from `styled-component`
3. Check if it has been called with the case template.

What do you think about this?

Reply:
Sounds like more work than I can invest at the moment ; ). Gonna try looking into it, soon, anyhow.